paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/AppearanceBottomSheetDialogFragment.kt

```diff
@@ -570,8 +570,10 @@ private fun Typography(
     FontScaleSlider(currentAppearance.typography.sizeScaleFactor) {
         updateAppearance(
             currentAppearance.copy(
-                typography = currentAppearance.typography.copy(
-                    sizeScaleFactor = it
+                typography = PaymentSheet.Typography(
+                    sizeScaleFactor = it,
+                    fontResId = currentAppearance.typography.fontResId,
```

Comment:
We're going to make typography internal properties internal, so we'll need to copy the pattern of AppearanceStore.State before we do that ticket.